dask_cuda/cli.py

```diff
@@ -5,25 +5,62 @@
 import click
 from tornado.ioloop import IOLoop, TimeoutError
 
-from dask import config
+from dask import config as dask_config
+from distributed import Client
 from distributed.cli.utils import install_signal_handlers
 from distributed.preloading import validate_preload_argv
 from distributed.security import Security
 from distributed.utils import import_term
 
-from ..cuda_worker import CUDAWorker
+from .cuda_worker import CUDAWorker
+from .utils import print_cluster_config
 
 logger = logging.getLogger(__name__)
 
 
 pem_file_option_type = click.Path(exists=True, resolve_path=True)
-
-
-@click.command(context_settings=dict(ignore_unknown_options=True))
-@click.argument("scheduler", type=str, required=False)
-@click.argument(
+scheduler = click.argument("scheduler", type=str, required=False)
+preload_argv = click.argument(
     "preload_argv", nargs=-1, type=click.UNPROCESSED, callback=validate_preload_argv
 )
+scheduler_file = click.option(
+    "--scheduler-file",
+    type=str,
+    default=None,
+    help="""Filename to JSON encoded scheduler information. To be used in conjunction
+    with the equivalent ``dask-scheduler`` option.""",
```

Comment:
```suggestion
    with the equivalent ``dask scheduler`` option.""",
```

Reply:
Not absolutely sure, but I imagine the hyphen here is also not necessary anymore (or deprecated)?

Reply:
Good catch! Yeah I can go through and switch out instances of the old Dask CLI for the new.

Reply:
I _think_ @jacobtomlinson had done it all (or almost all), so I hope there won't be many more.